spec/models/like_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  it 'is valid with a user and post' do
+    user = User.create(name: 'Liker')
+    post = Post.create(title: 'Test Post', text: 'This is a test post.', author: user)
+    like = Like.new(user:, post:)
+    expect(like).to be_valid
+  end
+
+  it 'is invalid without a user' do
+    post = Post.create(title: 'Test Post', text: 'This is a test post.')
+    like = Like.new(post:)
+    expect(like).to_not be_valid
+  end
+
+  it 'is invalid without a post' do
+    user = User.create(name: 'Liker')
+    like = Like.new(user:)
+    expect(like).to_not be_valid
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=equal%20to%20zero.-,Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.,-Need%20a%20big), Please add unit tests for the `update_likes_counter` method.